fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraSearch.java

```diff
@@ -105,7 +108,21 @@ public Response doSearch(@QueryParam(value = "condition") final List<String> con
                 }
             }
 
-            final var params = new SearchParameters(parsedFields, conditionList, maxResults, offset);
+            final Condition.Field orderByField;
+            try {
+                orderByField = Condition.Field.fromString(orderBy);
+            } catch (final Exception e) {
+                throw new InvalidQueryException("The order_by field must contain a valid value such as " +
+                        StringUtils.join(Condition.Field.values()));
+            }
+
+            if (!(order.equalsIgnoreCase("asc") || order.equalsIgnoreCase("desc"))) {
+                throw new InvalidQueryException("The order field is invalid:  valid values are \"asc\" and \"desc\"" +
+                        StringUtils.join(Condition.Field.values()));
```

Comment:
There are two valid values for the `order` field: asc and desc.
What is the purpose of listing all of the potential fields? ...likely should be removed.